modules/ROOT/pages/index.adoc

```diff
@@ -16,8 +16,6 @@ Refer to the DataStax Luna Streaming https://github.com/datastax/release-notes/b
 
 In addition to the distribution of https://pulsar.apache.org/en/versions/[Apache Pulsar {pulsar-version}], DataStax Luna Streaming provides:
```

Comment:
Do we want to cover other components of the DS LS?
* AdminConsole
* HeartBeat
* Burnell
* etc.

If so, these components have the biggest difference regarding deployment between K8s and tarball